<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PodnekoMainWindow</class>
 <widget class="QMainWindow" name="PodnekoMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Podneko v0.001 Pre-Alpha</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,1,10,1">
    <item>
     <widget class="QLabel" name="feedsLabel">
      <property name="text">
       <string>Feeds:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="feedsList">
      <item>
       <property name="text">
        <string>http://feeds.ign.com/ignfeeds/podcasts/beyond/</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>http://wtfpod.libsyn.com/rss</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="feedData">
      <property name="text">
       <string>Feed Data:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="feedTree">
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="grabFeed">
        <property name="text">
         <string>Refresh Feeds</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
